src/Objs/Product/ProductComponent.tsx

```diff
@@ -286,7 +286,7 @@ const LOCALIZERS = {
   en: {
     cartAddedMessage: 'Added __product__ to cart.',
     cartAddLabel: 'Add to cart',
-    cartRemovedMessage: 'Removed __product__ from cart.',
+    cartRemovedMessage: 'Removed item from cart.',
```

Comment:
* [ ] FYI this actually decreases usability. From the screenshots you can see how clear the add/remove messages are if the product name is included. Ideally we would always show the product name instead of never.

Reply:
> Ideally we would always show the product name instead of never.

But as far as I know you don't have access to the product name in the `DataDeleteButtonWidget` scenario.

Reply:
I tried adding a copy of the title of the product to `CartItem`.

With that I could use the following delete message: `Removed __CartItem.title__ from cart.`. If I render this in a `Scrivito.ContentTag` all is fine. But if I try to read it via `const deletedMessage = widget.get('deletedMessage')` I don't get the resolve value, but the raw `Removed __CartItem.title__ from cart.`.

Any idea how I can get a string with the resolved value from Scrivito?

Reply:
This can already be done by moving the delete button into the Product data group and wrapping it by a CartItem data group. With this setup, `Removed __Product.title__ from cart.` works.

Reply:
👍 for `useResolvedHtmlValue`!

Adressed in https://github.com/Scrivito/scrivito-portal-app/pull/722.